nox/popen.py

```diff
@@ -16,13 +16,11 @@
 import sys
 
 
-def popen(args, env=None, silent=False):
-    stdout = None
-
-    if silent:
+def popen(args, env=None, silent=False, stdout=None, stderr=subprocess.STDOUT):
```

Comment:
What happens when both `silent` and `stdout` are specified? What happens when the command fails but `stdout` isn't `PIPE`? Can we add test cases for that?